toolset/benchmark/test_types/db_type.py

```diff
@@ -19,7 +19,8 @@ def get_url(self):
         return self.db_url
 
     def verify(self, base_url):
-        '''Ensures body is valid JSON with a key 'id' and a key 
+        '''
```

Comment:
I'm not a fan of this style of comment, but it seems like we switch between this and `#`. Is there a reason/standard?

Reply:
This is the way python wants you to document your functions.